docs/controls/wpf-winforms/WebView.md

```diff
@@ -140,6 +140,9 @@ To set the initial content of the the **WebView** control, you can set the [Sour
 webView1.Navigate("http://www.contoso.com");

```

Comment:
Should we update the example code here to show that it's in a loading event?

Reply:
good call. I'll update